grease-cli/tests/x86/memshape-ptr-offset.x64.cbl

```diff
@@ -11,9 +11,7 @@
   (start start:
     (let rdi (get-reg rdi regs))
     (let rdi-ptr (pointer-read (Ptr 64) le rdi))
-    (let offset (bv 64 8))
-    (let rdi-ptr-offset (pointer-add rdi-ptr offset))
-    (let rdi-ptr-offset-contents (pointer-read (Bitvector 64) le rdi-ptr-offset))
+    (let rdi-ptr-offset-contents (pointer-read (Bitvector 64) le rdi-ptr))
```

Comment:
I'm surprised to see this test change - Was it previously incorrect?

Reply:
Yes, resolved in mattermost this was due to the missing store of the ptr